maestro-scalding/src/main/scala/au/com/cba/omnia/maestro/task/MaestroConfig.scala

```diff
@@ -69,6 +69,8 @@ case class MaestroConfig(
   lazy val connString      = args("jdbc")
   lazy val username        = args("db-user")
   lazy val password        = Option(System.getenv("DBPASSWORD")).getOrElse("")
+  lazy val envInputPrefix  = args.optional("env-input-prefix").getOrElse(args("env"))
```

Comment:
I think we can’t use “env” as the default because it will be a full environment like “physical.logical” (“datascience.dev”). Instead, I think we can use mandatory args (because envInputPrefix and envOutputPrefix are lazy vals: they are themselves optional and only accessed if needed):
```suggestion
  lazy val envInputPrefix  = args("env-input-prefix")
```